ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,338 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
```

Comment:
We're missing the standard future imports here. Copy these from any existing file.